CUJExecutor/app/src/androidTest/java/com/example/firstapp/ShellUtility.java

```diff
@@ -582,4 +582,18 @@ public void executeCUJ(String preStr, String postStr, int iterations, boolean re
             logData(allActionStamps);
         }
     }
+
+
+    public void dontExecuteTerminationAction(String preStr, String cujStr) throws Exception {
```

Comment:
Name doesn't accurately describe what it's doing.  It looks like it's doing a cached run of actions-1.